src/monmove.cpp

```diff
@@ -1117,6 +1117,12 @@ bool monster::move_to( const tripoint &p, bool force, const float stagger_adjust
     if( has_flag( MF_DRIPS_NAPALM ) ) {
         if( one_in( 10 ) ) {
             g->m.add_item_or_charges( pos(), item( "napalm" ) );
+            if( one_in( 4 ) ) {
+                g->m.add_field( pos(), fd_fire, 2 );
+                add_msg( m_warning,
```

Comment:
There needs to be a check that the player can actually see this before you print a message.